backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/AccountSheet_Report.sql

```diff
@@ -69,11 +54,46 @@ FROM de_metas_acct.RV_AccountSheet t
 WHERE TRUE
   AND t.PostingType = 'A'
   AND t.C_AcctSchema_ID = p_C_AcctSchema_ID
-  AND t.account_id = p_Account_ID
+  AND t.account_id = ANY((array_retain_positive(p_Account_IDs)))
   AND (p_DateAcctFrom IS NULL OR t.dateacct >= p_DateAcctFrom)
   AND (p_DateAcctTo IS NULL OR t.dateacct <= p_DateAcctTo)
 ORDER BY t.dateacct, t.ad_table_id, t.record_id, t.fact_acct_id
     ;
 $BODY$
 ;
 
+CREATE OR REPLACE FUNCTION de_metas_acct.AccountSheet_Report(
```

Comment:
u don't need overriding the function.
* make sure the function is always called with named parameters
* simple accept both parameters , have a default value of null for them
* in the function body, check which param is provided and use it. as an example (not perfect!) take a look at "de_metas_acct".product_costs_recreate, p_M_Product_ID and p_M_Product_IDs params